### PR TITLE
Move all Tuple Built-in functions to packages and some of the Int, Float and Char functions

### PR DESCRIPTION
**Pull request by OceanOak**
Changelog:

```
Area of change
- Move all Tuple2 and Tuple3 Built-in functions to packages
- Move some of the Int, Float and Char built-in functions to packages
```

### CODE REVIEW

**Comment by pbiggar** [comment]
Nice!

File: packages/darklang/stdlib/tuple2.dark [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+
+      /// Transform both values in a pair
+      let mapBoth
+        (fnFirst: 'a -> 'c)
+        (fnSecond: 'b -> 'd)
+        (tuple: Tuple<'a, 'b>)
+        : Tuple<'c, 'd> =
+        let (first, second) = tuple
+        ((first |> fnFirst), (second |> fnSecond))
```
Comment (StachuDotNet) [inline packages/darklang/stdlib/tuple2.dark:45]: this feels like over-use of pipes to me

compare:
- `(fnFirst first, fnSecond second)`
- `((first |> fnFirst), (second |> fnSecond))`

Comment (OceanOak) [reply packages/darklang/stdlib/tuple2.dark:45]: I think we can't do this yet, waiting for EApply changes

Comment (StachuDotNet) [reply packages/darklang/stdlib/tuple2.dark:45]: ah, you're right

**Pull request merged**